docs/conf.py

```diff
@@ -26,6 +26,7 @@
 
 extensions = [
     'sphinx.ext.todo',
+    'sphinx.ext.autodoc',
```

Comment:
We probably don't need this since not using any text from docstrings...

Reply:
True, may remove eventually!